SimGeneral/MixingModule/python/fullMixCustomize_cff.py

```diff
@@ -41,5 +41,24 @@ def setCrossingFrameOn(process):
         'TrackerHitsTIDLowTof', 
         'TrackerHitsTOBHighTof', 
         'TrackerHitsTOBLowTof')
+        
+    from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
+    run2_GEM_2017.toModify( process.mix.mixObjects,
+        mixSH = dict(
+            crossingFrames = process.mix.mixObjects.mixSH.crossingFrames + [ 'MuonGEMHits' ]
+        )
+    )
+    from Configuration.Eras.Modifier_run3_GEM_cff import run3_GEM
+    run3_GEM.toModify( process.mix.mixObjects,
+        mixSH = dict(
+            crossingFrames = process.mix.mixObjects.mixSH.crossingFrames + [ 'MuonGEMHits' ]
+        )
+    )
+    from Configuration.Eras.Modifier_phase2_muon_cff import phase2_muon
+    phase2_muon.toModify( process.mix.mixObjects,
```

Comment:
Since `run3_GEM` is included in the Phase2 eras, I'm concerned that having the same customization in `phase2_muon` will lead to a duplication...